runbot_coverage/models/runbot_repo.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import sys
+from openerp import api, fields, models
+
+
+class RunbotRepo(models.Model):
+    _inherit = 'runbot.repo'
+
+    use_coverage = fields.Boolean('Coverage', default=False)
+    coverage_config = fields.Char(
+        'Configuration file',
+        help='Your coveragerc relative to the build path.')
+    coverage_command = fields.Char(
```

Comment:
If is a bad idea use it then, Should we delete the `binary-dependency`?
Because the `odoo.tools.which` is used but without use this field to find the alternative path.
Then odoo won't allow us install it
But if is found and installed then for what we need this additional field?
Could you help me to understand?